libc/stdio_file.c

```diff
@@ -223,6 +249,7 @@ TEST(stdio_getput, fwritefread_basic)
 		TEST_ASSERT_EQUAL_INT(EOF, fgetc(filep));
 	}
 	TEST_ASSERT_EQUAL_INT(0, fclose(filep));
+	filep = NULL;
```

Comment:
Yeah, but because of cases where we have to set filep2 I would rather not use separate function, just sth like that:

```C
	ret = fclose(filep);
	filep = NULL;
	filep2 = NULL;

	TEST_ASSERT_EQUAL_INT(0, ret);
```